### PR TITLE
Add live inverse demo and include it in pa11y demo

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.0.1 to df6b4daaf335257078a83f1151856853cdff59f3
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.0.1 to 78439588468bfc77712510cad641bf60f8ef4a75
No significant bundle size differences found.

File: demos/src/pa11y.mustache [hunk 1]
```diff
@@ -787,7 +787,7 @@
 </div>
 
 <div class="demo-container demo-container--inverse">
-	<div class="o-teaser o-teaser--large o-teaser--inverse" data-o-component="o-teaser">
```
Comment (chee) [inline demos/src/pa11y.mustache:790]: does this replace the non-live one? is that a problem, i don't know? what do they look like

**Review by chee — APPROVED** [review]

File: demos/src/pa11y.mustache (hunk 1 above)
Comment (JakeChampion) [reply demos/src/pa11y.mustache:790]: non live is above this.

The non-live inverse was included in pa11y demo twice, probably one was meant to be inverse+live but we forgot to add the live modifier

**Comment by notlee** [comment]
This isn't particularly useful because the inverse demo is on black, but the inverse modifier is also often used on slate which offers less contrast. It might be more useful to have a new slate inverse demo though :) We could also include the live tag on both

File: demos/src/pa11y.mustache (hunk 1 above)
Comment (chee) [reply demos/src/pa11y.mustache:790]: ah!!

**Pull request closed without merging**